spec/models/article_spec.rb

```diff
@@ -62,11 +30,11 @@
 
     it 'must have unique titles' do
       article = Article.new({
-                    title: 'Monkey',
-                    body: 'Chicken',
-                    page: 1,
-                    author_name: 'Mike',
-                  })
+                              title: 'Monkey',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.